GeneratorInterface/GenFilters/src/MCParticlePairFilter.cc

```diff
@@ -230,7 +230,7 @@ bool MCParticlePairFilter::filter(edm::StreamID, edm::Event& iEvent, const edm::
 	     charge1 = charge((*p)->pdg_id());
 	     //totmomentum = momentum1 + typeApassed[i]->momentum();
```

Comment:
Remove commented code if not needed. Fix identation

Reply:
Hello @vargasa I can but that was not my fault :-)